content/en/docs/attestation/installation/kubernetes.md

```diff
@@ -31,6 +31,18 @@ NAME                                                   READY   STATUS    RESTART
 trustee-operator-controller-manager-77cb448dc-7vxck    1/1     Running   0          11m
 ```
 
+#### Override the trustee image
+
+By default, the operator deploys a trustee v0.11.0-based image, which can be overridden with an updated version (e.g. trustee v0.12.0).
```

Comment:
could we write this so that users can check what the default is and then explain how to change the default. I guess this sentence gets outdated sooner than later?